### ISSUE TITLE
eqtohomot

### ISSUE THREAD

**Issue by vladimirias** [UniMath/UniMath#585]
What if we make eqtohomot a coercion to Funclass?

**Comment by DanGrayson** [comment]
Coq objects, but it can be this:
```
Coercion eqtohomot : paths >-> homot.
```
But it's used only in `Ktheory`.
Perhaps you meant this:
```
Coercion toforallpaths : paths >-> homot.
```

We could take the same opportunity to make two more arguments of `toforallpaths` implicit, and then eliminate one of these two in favor of the other.  If we prefer `eqtohomot`, then the following identifiers should also be changed:
```
toforallpaths isweqtoforallpathsStatement weqtoforallpathsStatement
isweqtoforallpathsUAH isweqtoforallpathsAxiom isweqtoforallpaths
weqtoforallpaths toforallpaths_induction isaprop_isweqtoforallpathsStatement
toforallpaths_funextsec
```

**Comment by siddharthist** [comment]
I think `eqtohomot` should be removed in favor of `toforallpaths`, or the reverse. It can be confusing, for instance,
```
SearchAbout eqtohomot
```
brings up very little, while
```
SearchAbout toforallpaths
```
brings up a bunch of useful lemmas.

**Comment by DanGrayson** [comment]
I agree.  I prefer eqtohomot, though, because the name is more informative.

**Comment by mortberg** [comment]
I prefer `eqtohomot`, not so much because of the name but because it has the right number of implicit arguments.
